### ISSUE TITLE
why is pytorch image normalization not applied?

### ISSUE THREAD

**Issue by hweejuni** [lyuwenyu/RT-DETR#289]
Hi. Thank you for your great working.

I have a question on why you have not applied T.Normalize in preprocessing.
I can see T.Normalize is registered, but in 'dataloader.yml', Normalize is not included in 'transform' argument.
Is T.Normalize not applied in your experiments? or is it just that you applied in paddle and left 'Normalize' argument not included in 'dataloader.yml'? (which means in pretrained weight you indeed applied Normalize).
And furthermore, in your 'rtdetr_reader.yml' file, why is it normalized to [0,0,0] and [1,1,1], not ImageNet statistics?

**Comment by lyuwenyu** [comment]
应该影响不大， 保证推理和训练一致就行
